js/main.js

```diff
@@ -42,8 +42,80 @@ let result = 0;
 // 	}
 // }
 
-for (let i = 0; arr.length > i; i++) {
-	if (arr[i] % 2 === 0) {
-		console.log(arr[i]);
+// for (let i = 0; arr.length > i; i++) {
+// 	if (arr[i] % 2 === 0) {
+// 		console.log(arr[i]);
+// 	}
+// }
+//////////////////////////////////////////////////////////////
+////////////////////lesson-3//////////////////////////////////
+//////////////////////////////////////////////////////////////
+
+// const min = (a, x) => {
+// 	if (a > x) {
+// 		minResult = x;
+// 		return minResult;
+// 	} else if (a < x) {
+// 		minResult = a;
+// 		return minResult;
+// 	} else a == x;
+// 	minResult = "are equal";
+// 	return minResult;
+// };
+
+// const max = (a, x) => {
+// 	if (a < x) {
+// 		maxResult = x;
+// 		return maxResult;
+// 	} else if (a > x) {
+// 		maxResult = a;
+// 		return maxResult;
+// 	} else a == x;
```

Comment:
`else a == x`  is redundant condition 
`else ` - more clear

Reply:
and use `===` insted `==`

Reply:
fix